src/react-vrpc.js

```diff
@@ -41,7 +40,14 @@ export function createVrpcProvider ({
 class VrpcBackendMaker extends Component {
   constructor () {
     super()
-    this.state = { __global__: { vrpcIsLoading: true } }
+    this.state = {
+      initializing: true,
+      vrpc: { client: null, loading: true, error: null }
+    }
+    this.refresh = (key) => this.setState(prevState => {
```

Comment:
Unless I am missing some value in the constructor function I dont see a reason why this method is defined in here instead of a normal method. See example below

```javascript
  refresh = (key) => {
    this.setState(prevState => {
      const backend = { ...prevState[key][key].backend }
      return { [key]: { [key]: { ...prevState[key][key], backend } } }
    })
  }
```